docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1529,6 +1512,151 @@ simulants living in military group quarters we will set their employer
+
+International immigration
+'''''''''''''''''''''''''
+
+When sampling for immigration, there are three cases:
+
+#. We are sampling households to add in household moves.
+#. We are sampling GQ people to add in GQ person moves.
+#. We are sampling people in non-GQ households to add in non-reference-person moves.
+
+The details of the initial sampling are described in the immigration component documentation;
+here, we only consider how to perturb the PUMA values of an already-sampled entity (Row A),
+which has already been selected according to the 50% probability to have a perturbed
+PUMA.
+
+**80% of the time** (this probability is constant) we sample a new PUMA value from the same "immigration subset"
+of the same entity type in the ACS PUMS data.
+Specifically:
+
+#. If performing a household move, we sample the PUMA of another household with a recent
+   immigrant reference person, using household weights.
+#. If performing a GQ person move, we sample the PUMA of another GQ person who is a recent
+   immigrant, using person weights.
+#. If performing a non-reference-person move, we sample the PUMA of another non-GQ person
+   who is a recent immigrant and resides in a household where the reference person is not a
+   recent immigrant, using person weights.
+
+**The remaining 20% of the time**, we sample a new PUMA value from the same entity type, but without
+regard to immigration characteristics.
+Specifically:
+
+#. If performing a household move, we sample the PUMA of any household in the full ACS data, using household weights.
+#. If performing a GQ person move, we sample the PUMA of any GQ person in the full ACS data, using person weights.
+#. If performing a non-reference-person move, we sample the PUMA of any non-GQ person in the full ACS data, using person weights.
+
+As in population initialization, if creating a new household, the entire household is assigned the replacement PUMA value.
+
+By including the 20% probability of sampling from the full dataset, we ensure that immigration may
+occur in any PUMA, even if no ACS respondents were recent immigrants to that PUMA.
+Additionally, we rely less on the age, sex, race/ethnicity, etc joint distribution of recent immigrants specific to the PUMA
+in PUMAs where immigration is rarer (and we likely have smaller sample size to inform this distribution).
+
+Age
+^^^
+
+Households
+''''''''''
+
+After sampling a household to add to the simulation, whether at population initialization or
+from international immigration in a household move, the following steps are **always** performed.
+
+#. An age shift is generated by taking a random draw from a normal distribution with mean 0
+   and standard deviation 10 years.
+#. The age shift is added to the age values of all individuals in the household.
+#. If any age values in the household exceed 125 years, they are set to 125 years.
+#. Any individuals with negative age values are removed from the household.
```

Comment:
won't this result in losing babies a high percentage of the time (and young children a lower percentage of the time, etc)?

Reply:
Yes, and @aflaxman I think this is better than rounding to 0, which creates a spike at 0 in the age distribution.

I could see an argument for using truncation as I did in the individual case, based on the minimum age in the household, but I don't think it's obviously better.

One way to interpret this is as "what the household looked like X years ago" (assuming no moves, deaths, etc since then).

Reply:
@beatrixkh After discussion with Abie, we have switched to clipping the age to 0 when it is negative. I've updated the docs.